bgpd/bgp_evpn.c

```diff
@@ -4927,7 +4927,7 @@ void bgp_evpn_encode_prefix(struct stream *s, const struct prefix *p,
 		if (attr)
 			stream_put(s, &attr->esi, ESI_BYTES);
 		else
-			stream_put(s, 0, 10);
+			stream_put(s, zero_esi, 10);
```

Comment:
mmm - the raw `10` looks a little scary - could we use `sizeof` like the other instance, or use `ESI_BYTES`?

Reply:
I looked at `stream_put` again and it actually handles `NULL` pointer fine:
```
	if (src)
		memcpy(s->data + s->endp, src, size);
	else
		memset(s->data + s->endp, 0, size);
```
Sorry for the noise, going to close the PR.